modules/hcp-rn-4-17.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hosted-control-planes-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="hcp-rn-4-17_{context}"]
+= Release notes for {hcp} for {product-title} 4.17
+
+With this release, {hcp} for {product-title} 4.17 is available. {hcp-capital} for {product-title} 4.17 supports the {mce} version 2.7.
+
```

Comment:
@xenolinux - When I reviewed this yesterday, I forgot that we need to add a section for new features and enhancements here. I wrote a draft of the new section. Hopefully, it'll be easy to copy and paste into this file. I'm sure there's more to add. I added only the new features and enhancements that I worked on.

```
[id="hcp-4-17-new-features-and-enhancements_{context}"]
== New features and enhancements

This release adds improvements related to the following concepts:

[id="hcp-4-17-taints-tolerations_{context}"]
=== Custom taints and tolerations (Technology Preview)
For {hcp} on {VirtProductName}, you can now apply tolerations to hosted control plane pods by using the `hcp` CLI `-tolerations` argument or by using the `hc.Spec.Tolerations` API file. This feature is available as a Technology Preview feature. For more information, see xref:../hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc#hcp-virt-taints-tolerations_hcp-distribute-workloads[Custom taints and tolerations].

[id="hcp-4-17-tenancy-aws_{context}"]
=== Support for tenancy on {aws-short}
When you create a hosted cluster on {aws-short}, you can indicate whether the EC2 instance should run on shared or single-tenant hardware. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-aws-deploy-hc_hcp-deploy-aws[Creating a hosted cluster on {aws-short}].

[id="hcp-4-17-virt-dc_{context}"]
=== {hcp-capital} on {VirtProductName} in a disconnected environment is Generally Available
In this release, {hcp} on {VirtProductName} in a disconnected environment is Generally Available. For more information, see xref:../hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc#hcp-deploy-dc-virt[Deploying {hcp} on {VirtProductName} in a disconnected environment].

[id="hcp-4-17-multi_{context}"]
=== {hcp-capital} for an ARM64 {product-title} cluster on {aws-short} is Generally Available
In this release, {hcp} for an ARM64 {product-title} cluster on {aws-short} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-enable-arm-amd_hcp-deploy-aws[Running hosted clusters on an ARM64 architecture]

[id="hcp-4-17-ibm-z-ga_{context}"]
=== {hcp-capital} on {ibm-z-title} is Generally Available
In this release, {hcp} on {ibm-z-title} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc#hcp-deploy-ibmz[Deploying {hcp} on {ibm-z-title}].

[id="hcp-4-17-ibm-power-ga_{context}"]
=== {hcp-capital} on {ibm-power-title} is Generally Available
In this release, {hcp} on {ibm-power-title} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.adoc#hcp-deploy-ibmpower[Deploying {hcp} on {ibm-power-title}].
```

Reply:
Ah right! New features!! Thanks a lot for this draft. ❤️ I will check on other features to add.

Reply:
@xenolinux Thanks for adding the new features! I saw the errors from the bot. Should we move all the content into the assembly file? I notice that's what we do with the file for the OCP 4.17 release notes.

Reply:
yes. That would be accurate! Moved everything in assembly. I will re-check the preview and the PR for any other updates.